hydra.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: hydra
+  version: 2.3.0
+  epoch: 0
+  description: OpenID Certified OAuth 2.0 Server and OpenID Connect Provider
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - go
+  environment:
+    CGO_ENABLED: "1" # Needed for SQLite support
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/ory/hydra
+      tag: v${{package.version}}
+      expected-commit: ee8c339ddada3a42529c0416897abc32bad03bbb
+
+  - uses: go/install
+    with:
+      package: .
+      ldflags: -X 'github.com/ory/hydra/v2/driver/config.Version=$(git describe --abbrev=0 --tags)' -X 'github.com/ory/hydra/v2/driver/config.Date=$(TZ=UTC date -u "+%Y-%m-%dT%H:%M:%SZ")' -X 'github.com/ory/hydra/v2/driver/config.Commit=$(git rev-parse HEAD)'
+      tags: sqlite,json1,hsm
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}/usr/local/bin"
+      cp /home/build/melange-out/${{package.name}}/usr/bin/hydra "${{targets.destdir}}"/usr/local/bin/hydra
+
+update:
+  enabled: true
```

Comment:
We can also have probably ignoring regex pattern for releases that have `-rc` or `-pre` suffix

```bash
  ignore-regex-patterns:
    - rc*
    - pre*
  ```
  
  As it seems from their releases https://github.com/ory/hydra/releases